content/source/docs/enterprise/saml/identity-provider-configuration-okta.html.md

```diff
@@ -5,43 +5,51 @@ page_title: "SAML Okta Identity Provider Configuration - Terraform Enterprise"
+
+    | Okta Field                      | Terraform Enterprise SAML Field        | Value                                        |
+    | ------------------------------- | -------------------------------------- | -------------------------------------------- |
+    | **Single sign on URL**          | ACS Consumer (Recipient) URL           | `https://<TFE HOSTNAME>/users/saml/auth`     |
+    | **Use the SSO URL for Recipient URL and Destination URL**  (checkbox)  | | `enabled`                                    |
+    | **Audience URI (SP Entity ID)** | Metadata (Audience) URL                | `https://<TFE HOSTNAME>/users/saml/metadata` |
+    | **Name ID format** (drop-down)  |                                        | EmailAddress                                 |
+    | **Application username**        |                                        | Email                                        |
+
+    ~> **Note:** The SSO URL is used by the identity provider software during authentication. It is different from the [Login URL][login_url], which users visit in order to sign in to Terraform Enterprise.
+
+    The full name for the **Name ID format** in the SAML specification is `urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress`.
+
+    If you are using Terraform Enterprise version v201912-1 or later, you can also choose Email Prefix for the application username (this was unreliable in older versions due to a bug that required usernames to be at least 3 characters).
+
+1. Still in the **SAML Settings** section, optionally configure a site admin permissions attribute statement. This statement determines which users can administer the entire Terraform Enterprise instance (see [Administering Terraform Enterprise](../admin/index.html) for more information about site admin permissions). Under the **Attribute Statements (Optional)** header, configure a statement as follows:
+
+    | Field                       | Value      | Description    |
+    | --------------------------- | ---------- | -------------- |
+    | **Name**                    | `SiteAdmin` | This is the default name for TFE's site admin [attribute][]; the name of this attribute can be changed in [TFE's SAML settings](./configuration.html) if necessary. |
+    | **Name Format** (drop-down) | Basic      |                |
+    | **Value**                   |            | An [Okta expression](https://developer.okta.com/docs/reference/okta-expression-language/) that will evaluate to a boolean: `true` for every user who should have site admin permissions, but `false` for any users who should **not** have site admin permissions. The exact expression depends on the user properties you use to manage admin permissions. |
+
+
+1. Still in the **Configure SAML** page, configure a group attribute statement to report which teams a user belongs to. Under the **Group Attribute Statements (Optional)** header, configure the statement as follows:
```

Comment:
I don't think we need to say this part since we haven't indicated that the user has moved to another page :-) 
```suggestion
1. Configure a group attribute statement to report which teams a user belongs to. Under the **Group Attribute Statements (Optional)** header, configure the statement as follows:
```